liblepton/scheme/geda/object.scm

```diff
@@ -466,3 +468,9 @@
    (lambda (obj) (%mirror-object! obj x))
    objects)
   objects)
+
+
+
+( define-public object-locked?     %object-locked? )
+( define-public set-object-locked! %set-object-locked! )
```

Comment:
With regard to the `set-object-locked!` function, I would probably
rename it to `set-object-selectable!`, and add
`object-selectable?` instead of `object-locked?` (or together with
it).

Reply:
... and change the logic of the C function accordingly.